### ISSUE TITLE
On New Team, Validate invitation emails before hitting submit

### ISSUE THREAD

**Issue by jordanh** [ParabolInc/action#1311]
## Issue - Enhancement

When I am creating a new team and my trial has expired _or_ I am a paying customer and I want to invite a person who does not belong to my organization, I want to know that they are going to have to be approved before I hit submit, so that I am not surprised that I will have to wait and that I'll have an opportunity to nag them.

N.B. this will also reduce friction when folks can create new orgs under our freemium model. It may also serve as a useful prompt to create a new org when outsider don't really belong in the currently selected org.

The details:
- [ ] The organization dropdown menu has a Pro™ badge by the organizations that are Pro™.
- [ ] There is a radio button that defaults to selecting from an existing org where the dropdown menu defaults to top org by alpha.
- [ ] There is another radio button to create a new org. When users tap that button, they get a tooltip/hint box (TBD) that explains what a new org means.
- [ ] When coming from the Organization List under User Settings, the New Org radio button is selected.

The invitee by email field:
- [ ]  assumes that the user has chosen the right organization if at least one of the emails is already a member of the org.
- [ ] notifies the users with a hint box (TBD) that there are users who are not members and will require Billing Leader approval.
- [ ] notifies the user when there are no recognized user emails (for that org) and suggests maybe creating a new org
- [ ] check for emails that trigger hint on key up (and show the hint)
- [ ] mock up hint boxes for validation tips (approval, new org)
- [ ] bonus points: mock up hint box for duplicate team name
- [ ] bonus points: (perhaps separate issue) adding DraftJS and autosuggest menus that include existing org members as user is typing emails

**Estimated effort**: 13 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

**Comment by mattkrick** [comment]
kicking this to @ackernaut to show me what this should look like

**Comment by mattkrick** [comment]
breaking into smaller, separate issues.

**Comment by ackernaut** [comment]
Created 2 new issues:
#1427 Org dropdown has conditional Pro™ badge
#1426 Invitee by email field has the features in the checklist from this closed issue
